proxies/java/src/main/java/com/dvc_harness/proxy/controller/ProxyController.java

```diff
@@ -28,6 +30,29 @@ public Spec spec() {
         return new Spec();
     }
 
+    @PostMapping("/user")
+    public BaseResponse user(@RequestBody UserRequestBody body, HttpServletResponse response) {
+        try {
+            User sdkUser = User.builder()
+                .userId(body.user_id)
+                .name(body.name)
```

Comment:
does this work when these fields are null / aren't set?

Reply:
yep